webapp/app.py

```diff
@@ -147,10 +149,24 @@ def global_template_context():
 
     return {"version": VANILLA_VERSION, "versionMinor": version_minor, "path": flask.request.path}
 
+import markupsafe
+
+
+# requested_component =
+
+def class_reference(component=None):
+    with open("build/classreferences.yaml") as data_yaml:
```

Comment:
We can probably move it out from here to only read file when server starts instead of every request?